paimon-common/src/main/java/org/apache/paimon/TableType.java

```diff
@@ -28,8 +28,8 @@ public enum TableType implements DescribedEnum {
     TABLE("table", "Normal Paimon table."),
     FORMAT_TABLE(
             "format-table",
-            "A file format table refers to a directory that contains multiple files of the same format.");
-
+            "A file format table refers to a directory that contains multiple files of the same format."),
+    FLINK_MATERIALIZED_TABLE("flink-materialized-table", "A Flink materialized table.");
```

Comment:
It may should be a `materialized-table`. I am thinking the table created by Flink SQL can be refreshed by Spark SQL.